### PR TITLE
Mv'd sample tf files to dir, added README (its text needs work)

### PR DESCRIPTION
**Pull request by sehqlr**
This PR moves the sample files for Terraform into their own directory, including a README. The copy in the README needs work.

Will moving the files make things easier? Or is it unnecessary clutter?

### CODE REVIEW

**Comment by sehqlr** [comment]
This would fix #1030

**Comment by siddharthist** [comment]
This also needs to update every spot in the docs where the sample files are mentioned.

File: terraform/samples/README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Mantl Sample Terraform Configuration Files
+
+If you are just starting out using Mantl and terraform, here are some
+sample terraform configuration files that work with the other files
+that we've written here. All you need to do is `cp` the sample that
+matches with your provider to the root of the mantl directory, fill in
+your credentials, and you're all set!
```
Comment (siddharthist) [inline terraform/samples/README.md]: We should link to the actual documentation on these things (the getting started guide)

**Comment by sehqlr** [comment]
This is going to be superseded by dockerizing the deployment. I'm closing this

**Pull request closed without merging**